src/map/Pins.js

```diff
@@ -74,17 +75,22 @@
 
     Pins.prototype.layerEnter = function (base, svgElement, domElement) {
         Layer.prototype.layerEnter.apply(this, arguments);
-
+        
         this._pinsTransform = svgElement;
         this._selection.widgetElement(this._pinsTransform);
         this.pinsPaths = d3.select(null);
+        this
+            .tooltipHTML(function (d) {
+                return d.ext.tooltip === undefined ? "" : d.ext.tooltip;
```

Comment:
Could show the label if there was no specific tooltip?

Reply:
I don't think the GMapPin data has a label?

Reply:
...true...